src/Symfony/Component/DependencyInjection/Tests/ParameterBag/EnvPlaceholderParameterBagTest.php

```diff
@@ -119,6 +119,15 @@ public function testResolveEnvCastsIntToString()
         $this->assertSame('2', $bag->all()['env(int)']);
     }
 
+    public function testResolveEnvAllowsNull()
+    {
+        $bag = new EnvPlaceholderParameterBag();
+        $bag->get('env(NULL)');
```

Comment:
It's a bit confusing in this test to use `NULL` as the var name. What about replacing it by `NULL_VAR` ? (and `Null_Var` in the example below)